feelslikemonday/app/src/main/res/layout/activity_signup.xml

```diff
@@ -92,19 +92,19 @@
                 app:layout_constraintStart_toEndOf="@+id/signupPasswordView" />
```

Comment:
I think we should have a limit to the length of the password? All you have to do to set a max_length is go to this xml file, go on the 'design' view and click on the edit text you want to set a maximum length for (so user name and password). Then on the right hand side scroll down until you see the attribute "max_length" and just punch in a number! (we could do like 15 -ish??). 
Its super easy to do !